### PR TITLE
Add peak_to_peak mode to get_templates_amplitude

### PR DESCRIPTION
**Pull request by alejoe91**
@samuelgarcia @yger I think we need this option urgently for amplitudes!

### CODE REVIEW

File: src/spikeinterface/core/tests/test_template_tools.py [hunk 1]
```diff
@@ -56,8 +57,11 @@ def test_get_template_amplitudes(sorting_analyzer):
     peak_values = get_template_amplitudes(sorting_analyzer)
     print(peak_values)
     templates = _get_templates_object_from_sorting_analyzer(sorting_analyzer)
-    peak_values = get_template_amplitudes(templates)
-    print(peak_values)
+    peak_values = get_template_amplitudes(templates, abs_value=True)
+    print("peaks", peak_values)
```
Comment (h-mayorquin) [inline src/spikeinterface/core/tests/test_template_tools.py:61]: Let's remove the prints?

File: src/spikeinterface/core/tests/test_template_tools.py [hunk 2]
```diff
@@ -56,8 +57,11 @@ def test_get_template_amplitudes(sorting_analyzer):
     peak_values = get_template_amplitudes(sorting_analyzer)
     print(peak_values)
```
Comment (alejoe91) [inline src/spikeinterface/core/tests/test_template_tools.py:58]:
```suggestion
```

File: src/spikeinterface/core/tests/test_template_tools.py (hunk 1 above)
Comment (alejoe91) [inline src/spikeinterface/core/tests/test_template_tools.py:61]:
```suggestion
```

File: src/spikeinterface/core/tests/test_template_tools.py [hunk 3]
```diff
@@ -56,8 +57,11 @@ def test_get_template_amplitudes(sorting_analyzer):
...
     templates = _get_templates_object_from_sorting_analyzer(sorting_analyzer)
-    peak_values = get_template_amplitudes(templates)
-    print(peak_values)
+    peak_values = get_template_amplitudes(templates, abs_value=True)
+    print("peaks", peak_values)
+    peak_to_peak_values = get_template_amplitudes(templates, mode="peak_to_peak")
+    print("peak to peak", peak_to_peak_values)
```
Comment (alejoe91) [inline src/spikeinterface/core/tests/test_template_tools.py:63]:
```suggestion
```

File: src/spikeinterface/core/tests/test_template_tools.py (hunk 1 above)
Comment (alejoe91) [reply src/spikeinterface/core/tests/test_template_tools.py]: done :)

**Review by h-mayorquin — APPROVED** [review]
LGTM

**Review by samuelgarcia — APPROVED** [review]

**Pull request merged**